runtime/src/in_mem_accounts_index.rs

```diff
@@ -1003,7 +993,36 @@ impl<T: IndexValue> InMemAccountsIndex<T> {
                         evictions_random.push(*k);
                     }
                 }
-                Self::update_time_stat(&self.stats().flush_scan_update_us, m);
+                Self::update_time_stat(&self.stats().flush_scan_us, m);
+            }
+            {
+                // write to disk outside giant read lock
+                let m = Measure::start("flush_update"); // we don't care about lock time in this metric - bg threads can wait
+                for (k, v) in dirty_items {
+                    if v.dirty() {
+                        // already marked dirty again, skip it
+                        continue;
+                    }
+                    loop {
+                        let disk_resize = {
+                            let slot_list = v.slot_list.read().unwrap();
+                            disk.try_write(&k, (&slot_list, v.ref_count()))
+                        };
+                        match disk_resize {
+                            Ok(_) => {
+                                // successfully written to disk
+                                flush_entries_updated_on_disk += 1;
+                            }
+                            Err(err) => {
+                                // disk needs to resize. This item did not get resized. Resize and try again.
```

Comment:
> This item did not get resized.

Is this supposed to be something like "this item did not get written to disk" or "this item did not get flushed"?